client/src/components/LandingPage/UserActivity.jsx

```diff
@@ -0,0 +1,68 @@
+import Typography from '@mui/material/Typography';
+import Paper from '@mui/material/Paper';
+import theme from '../../theme';
+import Stack from '@mui/material/Stack';
+import MaterialLink from '@mui/material/Link';
+import { Link } from 'react-router-dom';
+
+const UserActivity = () => {
+	return (
+		<Stack
+			pt={{ xs: 4, sm: 12 }}
+			pb={{ xs: 8, sm: 16 }}
+			spacing={{ xs: 3, sm: 6 }}
+			direction={{ xs: 'column', sm: 'row' }}
+			justifyContent={'center'}>
+			<Stack
+				alignItems={'center'}
+				justifyContent={'center'}
+				spacing={2}>
+				<Typography variant="h6">You Are Reading</Typography>
+				<Paper
+					elevation={5}
+					sx={{
+						bgcolor: theme.palette.primary.main,
+						width: { xs: '150px', sm: '200px' },
+						height: { xs: '220px', sm: '300px' },
+					}}></Paper>
+				<MaterialLink component={Link}>
+					View all currently reading
```

Comment:
I think the landing page looks great, and I think we should keep the layout you created. Since we've decided to move forward with a single book list that doesn't categorize them, we might want a single link that says "View My Books" instead of the three separate ones. I think we should keep the display of three books that you have, but just grab the most recent additions to the user's book list. We could call the header something like "Recently added to my books". What do you think @jaredsina

Reply:
I agree, this will allow us to keep most of the work that has already been done on this component. I will move forward with changing it to recently added to my books and update the links!